sdk/mobile-center-distribute/src/main/java/com/microsoft/azure/mobile/distribute/Distribute.java

```diff
@@ -311,15 +314,14 @@ protected String getLoggerTag() {
 
     @Override
     public synchronized void onStarted(@NonNull Context context, @NonNull String appSecret, @NonNull Channel channel) {
-        super.onStarted(context, appSecret, channel);
         mContext = context;
         mAppSecret = appSecret;
         try {
             mPackageInfo = mContext.getPackageManager().getPackageInfo(mContext.getPackageName(), 0);
         } catch (PackageManager.NameNotFoundException e) {
             MobileCenterLog.error(LOG_TAG, "Could not get self package info.", e);
         }
-        resumeDistributeWorkflow();
+        super.onStarted(context, appSecret, channel);
```

Comment:
Yes because applyStateEnabled is now managed in abstract class, I fixed a bug where mPackageInfo was not set and applyStateEnabled called first.